config/i18n/locales/dothraki/translations.json

```diff
@@ -0,0 +1,94 @@
+{
+  "buttons": {
+    "forum": "Forum",
+    "donate": "Donate",
+    "load-more-articles": "Load More Articles"
+  },
+  "search": {
+    "label": "Search",
+    "accessible-name": "Submit your search query",
+    "placeholder": "Search 8,000+ tutorials",
+    "see-results": "See all results for {{ searchQuery }}",
+    "no-tutorials": "No tutorials found"
+  },
+  "banner": {
+    "default": "Learn to code — {{ <0> }}free 3,000-hour curriculum{{ </0> }}",
+    "authenticated": "Support our nonprofit and our mission. {{ <0> }}Donate to freeCodeCamp.org{{ </0> }}.",
+    "authenticated-donor": "Thank you for supporting freeCodeCamp through {{ <0> }}your donations{{ </0> }}."
+  },
+  "default-bio": "Read {{ <0> }}more posts{{ </0> }}.",
+  "author": {
+    "no-posts": "No posts",
+    "one-post": "1 post",
+    "multiple-posts": "{{ postCount }} posts"
+  },
+  "tag": {
+    "no-posts": "A collection of posts",
+    "one-post": "A collection of 1 post",
+    "multiple-posts": "A collection of {{ postCount }} posts"
+  },
+  "404": {
+    "page-not-found": "Page not found",
+    "go-to-front-page": "Go to the front page"
+  },
+  "social-row": {
+    "cta": {
+      "tweet-a-thanks": "If you read this far, tweet to the author to show them you care. {{ <0> }}Tweet a thanks{{ </0> }}",
+      "tweet-it": "If this article was helpful, {{ <0> }}tweet it{{ </0> }}."
+    },
+    "tweets": {
+      "default": "Thank you {{ author }} for writing this helpful article.",
+      "translation": "Thank you {{ author }} for writing this helpful article, and {{ translator }} for translating it."
+    }
+  },
+  "learn-to-code-cta": "Learn to code for free. freeCodeCamp's open source curriculum has helped more than 40,000 people get jobs as developers. {{ <0> }}Get started{{ </0> }}",
+  "original-author-translator": {
+    "roles": {
+      "author": "Author: {{ name }} ({{ locale }})",
+      "translator": "Translator: {{ name }}"
+    },
+    "details": {
+      "original-article": "{{ <0> }}Original article:{{ </0> }} {{ title }} by {{ author }}",
+      "translated-by": "{{ <0> }}Translated by:{{ </0> }} {{ translator }}"
+    },
+    "locales": {
+      "arabic": "Arabic",
+      "bengali": "Bengali",
+      "chinese": "Chinese",
+      "english": "English",
+      "espanol": "Spanish",
+      "italian": "Italian",
+      "japanese": "Japanese",
+      "portuguese": "Portuguese",
+      "swahili": "Swahili",
+      "urdu": "Urdu"
+    }
```

Comment:
@scissorsneedfoodtoo I am not sure why this config is here. I think this is creating a problem because there is a need for duplication in multiple files and is missed in some of the languages altogether. Do you know how to consolidate this?

Reply:
Sorry @raisedadead, just seeing this now.

Those strings are for the original author / translator feature to show the source article's publication language as a localized string for the each publication.

For example, if an English article is translated into Spanish, it will show up as ingles rather than English:

<img width="495" alt="image" src="https://user-images.githubusercontent.com/2051070/178901195-aa4dd6dc-fd97-4adc-9e09-4877443278fe.png">

Not sure if there's a great way to automate this, though I can think on it.

But since the vast majority of articles will likely translated from English, I wouldn't be opposed to removing these strings to keep things simple. It shouldn't be difficult to figure out which language the original article is in based on the title and links we insert before the translated article itself.

Also, I could see this being a burden for translators, especially once we start adding more publications.

Reply:
Thanks for sharing the context. IMHO we should just be fine by using the original English names for the languages and keep it simple.